articles/azure-functions/functions-scale.md

```diff
@@ -74,7 +74,7 @@ Consider the Azure Functions premium plan in the following situations:
 * You have a high number of small executions and have a high execution bill but low GB second bill in the consumption plan.
 * You need more CPU or memory options than what is provided by the Consumption plan.
 * Your code needs to run longer than the [maximum execution time allowed](#timeout) on the Consumption plan.
-* You require features that are only available on a Premium plan, such as VNET/VPN connectivity.
+* You require features that are only available on a Premium plan, such as VNet/VPN connectivity.
```

Comment:
Per guidelines, we are supposed to spell out `virtual network` every time 😦 
```suggestion
* You require features that are only available on a Premium plan, such as virtual network connectivity.
```
Probably we should get an edit pass for this in the rest of the Functions docs 😉

Reply:
@andresnava  If you can please accept my suggestion, we can merge your PR. Thanks.